pkgs/data/fonts/iosevka/default.nix

```diff
@@ -1,83 +1,60 @@
-{
-  stdenv, lib, pkgs,
-  fetchFromGitHub, fetchurl,
-  nodejs, ttfautohint-nox, otfcc,
+{ stdenv, lib, pkgs
+, nodejs, remarshal, ttfautohint-nox, otfcc
 
-  # Custom font set options.
-  # See https://github.com/be5invis/Iosevka#build-your-own-style
-  design ? [], upright ? [], italic ? [], oblique ? [],
```

Comment:
Why were these options removed?

Reply:
They can now live within `privateBuildPlan`, which is also more future-proof in case upstream adds more build options. Wrapping any existing overlays with `privateBuildPlan` should just work.

Reply:
Actually, you also have to specify the family... would this be a concern in terms of breaking changes?

Reply:
I tried this by copying the [example](https://github.com/be5invis/Iosevka#build-your-own-style) build plan, but the build failed.

Reply:
How did you copy it? It should still be in object format, not a string. I can add an example if you like.

Reply:
@ttuegel for reference, here's the overlay I used to test the functionality. You could also add the other options (upright, italic, oblique) and they would be converted.

```
iosevka = super.iosevka.override {
  set = "expanded";

  privateBuildPlan = {
    family = "Iosevka Expanded";

    design = [
      "sans"
      "expanded"
    ];
  };
};
```

Reply:
@ttuegel do you still have a problem with this? I could add something like this to the header comment to make things clearer

Reply:
Yes, please add a comment like this to the header, because it will break anyone's configuration who builds a custom Iosevka right now. Otherwise, let's get this merged ASAP, since Iosevka is broken in master anyway.